docs/releasenotes/release-1.136.md

```diff
@@ -0,0 +1,36 @@
+
+** This version is not yet released; this document is gathering release notes for the future release **
+
+*   Special shout-outs to @acpana, @anhdle-sso, @cheftako, @gemmahou, @jingyih, @justinsb, @xiaoweim, @yuwenma for their contributions to this release.
+
+## New Beta Resources (Direct Reconciler):
+
+*   [`AssetFeed`](https://cloud.google.com/config-connector/docs/reference/resource-docs/asset/assetfeed)
+*   [`BigQueryReservationAssignment`](https://cloud.google.com/config-connector/docs/reference/resource-docs/bigqueryreservation/bigqueryreservationassignment)
+*   [`CloudDeployDeliveryPipeline`](https://cloud.google.com/config-connector/docs/reference/resource-docs/clouddeploy/clouddeploydeliverypipeline)
+*   [`ComposerEnvironment`](https://cloud.google.com/config-connector/docs/reference/resource-docs/composer/composerenvironment)
+
+## New Fields:
+
+*   [`ComposerEnvironment`](https://cloud.google.com/config-connector/docs/reference/resource-docs/composer/composerenvironment)
+    *   Added `spec.storageConfig` field.
+    *   Added `spec.config.workloadsConfig.dagProcessor` field.
+    *   Added `spec.config.workloadsConfig.triggerer` field.
+    *   Added `spec.config.softwareConfig.webServerPluginsMode` field.
+    *   Added `spec.config.softwareConfig.cloudDataLineageIntegration` field.
+
+## Reconciliation Improvements:
+
+*   Introduce [Stateful Reconciliation for Direct Controllers](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/docs/designs/stateful-reconciliation-with-cookie.md). The controller will store a hash of the last successfully applied `.spec` in the resource's `.status`. This provides a lightweight, GitOps-safe historical record, allowing the controller to know with certainty when a user has modified their desired state. This enhanced reconciliation will be applied to the KRM-native controller gradually
+
+## Bug Fixes:
+
+*   Added support for checking etag in spec for alpha resources.
```

Comment:
Not blocking but it would be nice if the bug fixes had links to the relevant PR. (eg https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.34.md#bug-or-regression-2)